pkg/storage/registry/spaces/Readme.md

```diff
@@ -0,0 +1,52 @@
+# Spaces Registry
+
+The spaces registry recognizes individual spaces instead of storage providers.
+While it is configured with a list of storage providers, it will query them for all storage spaces and use the space ids to resolve id based lookups.
+Furthermore, path based lookups will take into account a path templvate to present a human readable file tree.
+
+## Configuration
+
+The spaces registry takes two configuration options:
+
+1.  `home_template` is used to buld a path for the users home. It uses a template that can access the current user in the context, e.g. `/users/{{.Id.OpaqueId}}`
+2.  `rules` is a map of path patterns to config rules
+
+### Patterns
+
+A pattern can be a simple path like `/users` or a regex like `/users/[0-9]`. It can also contain a template `/users/{{.CurrentUser.Id.OpaqueId}}/Shares`.
+The pattern is used when matching the path for path based requests.
+
+### Rules
+
+A rule has several properties:
+
+*   `mapping` unused?
```

Comment:
should we not remove TODO's from docs?

Reply:
Done in https://github.com/cs3org/reva/pull/2383